### PR TITLE
Support absolute paths to source code

### PR DESCRIPTION
**Pull request by laupow**
Relative paths inside the action are more common, but it should work for absolute paths also.

### CODE REVIEW

File: src/docker.ts [hunk 1]
```diff
@@ -57,11 +57,13 @@ export async function dockerBuildpackCompile(
 		warning(`Output directory ${destDir} already exists. Compile will overwrite firmware.bin if it exists.`);
 	}
 
+	info(`Compiling...`);
+	const inputDir = sources.charAt(0) === '/' ? sources : `${workingDir}/${sources}`;
```
Comment (hugomontero) [inline src/docker.ts:61]: there is a way to know if a path is absolute or relative:
By using `path` library:
`path.isAbsolute(myPath)`

Check this 👉  https://github.com/particle-iot/cli/blob/37d115e8a77ad24c7058a774eae734ccec39d48b/packages/cli/src/lib/reify-path.js#L12

Comment (hugomontero) [reply src/docker.ts:61]: This is another example: https://github.com/particle-iot/cli/blob/37d115e8a77ad24c7058a774eae734ccec39d48b/packages/toolchain-manager/src/dependencies/base.js#L242

In summary I think would be better to use `path.join` that will do the proper `/` `\` join depending on the OS so you don't need to worry about that.

Comment (laupow) [reply src/docker.ts:61]: Good ideas, I'll make changes to use `path` functions

**Review by hugomontero — APPROVED** [review]
Looks Great! 👍

**Pull request merged**